app/src/main/java/ir/ac/shirazu/softwareproject/server_api/Meal/MyKit.java

```diff
@@ -29,7 +29,7 @@
     private static void parseFoodInfo(String responseBody, Student student) {
 
         try {
-            FoodInfo foodInfo = new FoodInfo();
+//            FoodInfo foodInfo = new FoodInfo();
             JSONArray information = new JSONArray(responseBody);
             for (int i = 0; i < information.length(); i++) {
                 JSONObject album = information.getJSONObject(i);
```

Comment:
@hosseindehghanipour1998 why you didn't use foodInfo object? why you set Info to student object?